pyproject.toml

```diff
@@ -13,6 +13,7 @@ readme = "README.md"
 dependencies = [
     "ruamel.yaml ~=0.17",
     "psutil ~=5.9",
+    "snakemake ~=7.32"
```

Comment:
@jmtsuji This was probably because I failed to keep them synced up. I think I initially had the version more generic. We may need to change this in the future but we can do that then.

Reply:
@LeeBergstrand OK, sounds good to revisit this after we upgrade to snakemake 8!